spyder_notebook/widgets/dom.py

```diff
@@ -18,10 +18,7 @@ class DOMWidget(WebView):
 
     def __init__(self, parent):
         super(DOMWidget, self).__init__(parent)
-        if WEBENGINE:
-            self.page = self.page()
-        else:
-            self.page = self.page().mainFrame()
```

Comment:
Please don't remove this. This is very important to make things work for the packages provided by Anaconda and the PyQt5 wheels.